coding/python-imports.md

```diff
@@ -4,6 +4,33 @@ Python imports can be reorganized, sorted, and changed during development.
 
 Here are some best practices about Python imports.
 
+## Usage of absolute/relative imports
+
+Python language allow to import resources either using absolute path, either
+using path relative to current file. Guidelines for RERO-ILS project are 
+defined as follows :
+* Always give priority to **absolute** imports when import references  
+  resource/class/method outside the current module.
+* Use **relative** import for resources/classes/methods of the same module.
```

Comment:
Python allows importing of resources using a path that is either absolute, or relative to current file. The guidelines for RERO-ILS project are 
defined as follows:
* Always give priority to **absolute** paths when importing  
  resources/classes/methods from *outside* the current module.
* Use **relative** path when importing resources/classes/methods of the same module.